programs/voter-stake-registry/src/instructions/grant.rs

```diff
@@ -55,13 +57,17 @@ pub struct Grant<'info> {
     )]
     pub deposit_token: Box<Account<'info, TokenAccount>>,
 
+    #[account(mut)]
     pub authority: Signer<'info>,
 
     #[account(mut)]
     pub payer: Signer<'info>,
 
+    pub deposit_mint: Box<Account<'info, Mint>>,
```

Comment:
pretty sure the mint is unnecessary, can just use deposit_token.mint

Reply:
if it isn't unnecessary, there needs to be some kind of constraint that deposit_mint == deposit_token.mint